frontend/public/components/RBAC/bindings.jsx

```diff
@@ -224,9 +223,8 @@ class ListDropdown_ extends React.Component {
       }, state.items);
     });
 
-    const { selectedKey } = this.state;
     // did we switch from !loaded -> loaded ?
-    if (!this.props.loaded && !selectedKey) {
+    if (!this.props.loaded || !selectedKey) {
```

Comment:
I'm not sure this change is correct?

Reply:
I think this was a bug.  The original code made it so the title (placeholder) could never go back to unselected after the page loaded.  As part of my clear option, I need the title to go back to the default state which would never happen since the loaded flag was always true.